### PR TITLE
Command line console tool

### PR DESCRIPTION
**Pull request by hyades**

### CODE REVIEW

File: console.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        c.establish_connection()
+        if len(line.split()) != 4:
+            print 
+        print c.adjust_pip(*map(int, line.split()))
+
+    def do_switch(self, line):
+        try:
+            val = self.SWITCH_MAPPING[line.lower()]
+        except KeyError:
+            print "Invalid"
```
Comment (mithro) [inline console.py]: Need a return here.

File: console.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        print "Valid modes - {0}".format(", ".join([i for i in self.COMPOSITE_MAPPING]))
+
+    def complete_set_composite_mode(self, text, line, begidx, endidx):
+        return [i for i in self.COMPOSITE_MAPPING if i.startswith(text)]
+
+    def do_adjust_pip(self, line):
+        c = Controller()
+        c.establish_connection()
+        if len(line.split()) != 4:
+            print
```
Comment (mithro) [inline console.py]: Probably a return here too.

File: console.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    COMPOSITE_MAPPING = {
+        'none': Controller.COMPOSITE_NONE,
+        'pip': Controller.COMPOSITE_PIP,
+        'dual_preview': Controller.COMPOSITE_DUAL_PREVIEW,
+        'preview': Controller.COMPOSITE_DUAL_PREVIEW,
+        'equal': Controller.COMPOSITE_DUAL_EQUAL,
+        'dual_equal': Controller.COMPOSITE_DUAL_EQUAL
+    }
+
+    SWITCH_MAPPING = {
```
Comment (mithro) [inline console.py]: This names have changed.

Comment (hyades) [inline console.py]: done

File: console.py (hunk 1 above)
Comment (hyades) [inline console.py]: done

**Pull request merged**